Libplanet.Tools/download.mjs

```diff
@@ -0,0 +1,145 @@
+// The planet binary downloader for npm.
+// Inspired by Elm's npm packaging: <https://www.npmjs.com/package/elm>.
+"use strict";
+const nodeMajorVersion = process.versions.node.split('.')[0];
+if (nodeMajorVersion < 12) {
+  throw new Error("Requires Node.js v12 or Higher Version.");
+}
+
+import child_process from "child_process";
+import fs from "fs";
```

Comment:
It seems like the code is using the Promise variant exclusively. Directly importing from `fs/promises` might be preferable.

```suggestion
import { readFile, mktemp, copyFile, chmod } from "fs/promises"; // Don't forget to replace fs.promises in the same file!
```

Reply:
Applied: 656979cbc2ec6f3c9a7789bcebe5380ed4139c14.